daceml/onnx/shape_inference/symbolic_shape_infer.py

```diff
@@ -21,19 +21,26 @@ def get_attribute(node, attr_name, default_value=None):
 
 
 def get_dim_from_type_proto(dim):
-    return getattr(dim, dim.WhichOneof('value')) if type(dim.WhichOneof('value')) == str else None
+    return getattr(dim, dim.WhichOneof('value')) if type(
```

Comment:
Please revert formatting changes in this file: we don't maintain it, it's just mirrored from ONNX Runtime